src/Butil/Bit.Butil/Scripts/userAgent.ts

```
@@ -0,0 +1,595 @@
+            'webOS',
+            'Tablet OS',
+            'Tizen',
+            'Linux',
+            'Mac OS X',
+            'Macintosh',
+            'Mac',
+            'Windows 98;',
+            'Windows '
+        ]);
+
+        layout && (layout = [layout]);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)